src/bespokelabs/curator/status_tracker/batch_status_tracker.py

```diff
@@ -0,0 +1,106 @@
+import logging
+
+from pydantic import BaseModel, Field
+from bespokelabs.curator.types.generic_batch import GenericBatch
+
+logger = logging.getLogger(__name__)
+
+
+class BatchStatusTracker(BaseModel):
+    n_total_requests: int = Field(default=0)
```

Comment:
Why do we need this `n_` here and elsehwere?

Reply:
We are using it to differentiate between counts and objects. 
E.g. 
BatchStatusTracker.n_submitted_batches vs BatchStatusTracker.submitted_batches